packages/bigcommerce/src/api/utils/fetch-graphql-api.ts

```diff
@@ -7,7 +7,7 @@ const fetchGraphqlApi: (getConfig: () => BigcommerceConfig) => GraphQLFetcher =
   async (
     query: string,
     { variables, preview } = {},
-    options: { headers?: HeadersInit } = {}
+    headers?: HeadersInit
```

Comment:
I don't see this change as a good idea, having an options object that includes headers is far better IMO.

Reply:
This was aligned with the changes already **merged**, so back then you missed this change on the previous PR. I reverted these changes to reflect your comment with all providers to have headers passed in an options obj.